tests/foreman/ui/test_ldap_authentication.py

```diff
@@ -1186,6 +1016,42 @@ def test_single_sign_on_using_rhsso(enable_external_auth_rhsso, session):
         update_rhsso_settings_in_satellite(revert=True)
 
 
+@upgrade
+@destructive
+def test_session_expire_rhsso_idle_timeout(session):
+    """Verify the idle session expiration timeout with external authentication RH-SSO
+
+    :id: 80247b30-a988-11ea-943c-d46d6dd3b5b2
+
+    :steps:
+        1. Change the idle timeout settings for the External Authentication
+        2. Login into Satellite using RHSSO login and wait for the idle timeout
+
+    :expectedresults: After completion of the idle timeout user session
+        should get expired
+    """
+    try:
+        update_rhsso_settings_in_satellite()
+        property_name = 'idle_timeout'
+        setting_entity = entities.Setting().search(query={'search': f'name={property_name}'})[0]
+        setting_entity.value = 1
+        setting_entity.update({'value'})
+
+        with session(login=False):
+            session.rhsso_login.login(
+                {'username': settings.rhsso.rhsso_user, 'password': settings.rhsso.password}
+            )
+            sleep(360)
```

Comment:
Instead of Hard wait time can we use `wait_for` ?

Reply:
https://github.com/SatelliteQE/robottelo/pull/7832#issue-434630704 mentioned here about the details.